package.json

```diff
@@ -41,7 +41,7 @@
     "@nestjs/testing": "6.11.8",
     "@semantic-release/changelog": "5.0.0",
     "@semantic-release/git": "9.0.0",
-    "@types/amqp-connection-manager": "2.0.8",
+    "@types/amqp-connection-manager": "^2.0.8",
```

Comment:
renovate(dependencies monitor service will pin dependencies with auto-created PR), please not use `^`

Reply:
```suggestion
    "@types/amqp-connection-manager": "2.0.8",
```